common/src/main/java/org/apache/drill/common/logical/FormatPluginConfigBase.java

```diff
@@ -27,11 +27,20 @@
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(FormatPluginConfigBase.class);
 
 
-  public synchronized static Class<?>[] getSubTypes(DrillConfig config){
-    List<String> packages = config.getStringList(CommonConstants.STORAGE_PLUGIN_CONFIG_SCAN_PACKAGES);
-    Class<?>[] sec = PathScanner.scanForImplementationsArr(FormatPluginConfig.class, packages);
-    logger.debug("Adding Format Plugin Configs including {}", (Object) sec );
-    return sec;
+  public synchronized static Class<?>[] getSubTypes(DrillConfig config) {
+    List<String> packages =
+        config.getStringList(CommonConstants.STORAGE_PLUGIN_CONFIG_SCAN_PACKAGES);
+    Class<?>[] pluginClasses =
+        PathScanner.scanForImplementationsArr(FormatPluginConfig.class, packages);
+    if (logger.isDebugEnabled()) {
+      final StringBuilder sb = new StringBuilder();
```

Comment:
I don't have a strong recommendation for where it would belong, but as long as this code is in 3 places in this patch, and it seems like it would be useful for other cases, we should probably make it available in a static utility class.